packages/fluentui/ability-attributes/schema.json

```diff
@@ -64,6 +64,48 @@
         }
       ]
     },
+    "$setsize": {
+      "name": "setsize",
+      "attributes": [
+        {
+          "name": "aria-setsize",
+          "constraints": [
+            {
+              "xpath": "number(@aria-setsize) = @aria-setsize",
+              "description": "aria-setsize attribute value cannot be empty"
```

Comment:
Let's do «should be a number» instead of «cannot be empty» for all three of setsize, posinset and level.

Reply:
Done